oop2-encapsulate-inherit.js

```diff
@@ -5,27 +5,269 @@
     - Publisher class.  It should have an authors and books property.
     - Review class.  It should have a rating and user property.
     -Prevent the properties from being accessed by outside code, using the encapsulation concept of "private variables".  Create methods that return the values of these private variables in case other pieces of the code need to access the data of the 4 classes.  Also create methods that allow outside code to "set" new values for the properties on the classes, without directly manipulating the values themselves. 
-
+*/
     //your code here...
 
+    class Book {
+        #author;
+        #publisher;
+
+        constructor(authorm, publusher){
+            this.#author = author;
+            this.#publisher = publisher;
```

Comment:
Here, and throughout the file, you don't need to use `#` every time you reference the variable. In fact it's much better not to. The `#` just needs to be on the field declaration